docs: update `Suspense.md` (#1125)
reactjs/ko.react.dev: src/content/reference/react/Suspense.md

### `<Suspense>` {/*suspense*/}

#### Props {/*props*/}

> * `children`: 궁극적으로 렌더링하려는 실제 UI입니다. `children`의 렌더링이 지연되면, Suspense는 `Fallback`을 대신 렌더링합니다.

Reviewer: ```suggestion
* `children`: 궁극적으로 렌더링하려는 실제 UI입니다. `children`의 렌더링이 지연되면, Suspense는 `fallback`을 대신 렌더링합니다.
```

백틱(<code>`</code>)은 마크다운 문서에서 인라인 코드 블럭을 나타내는 기호로, 코드와 관련된 내용이 본문에 나올 경우 사용됩니다.

이때, 백틱안에 들어 있는 `fallback`은 코드상의 내용을 나타내므로 그대로 `fallback`으로 표시해야 합니다.

백틱으로 묶여있지 않은 일반적인 Fallback 이라는 용어만 대문자로 바꿔주셔야 합니다!

아래도 모두 마찬가지입니다.

Author: 수정사항 반영했습니다!
번거롭게 해드려서 죄송합니다.

> - [Relay](https://relay.dev/docs/guided-tour/rendering/loading-states/)와 [Next.js](https://nextjs.org/docs/getting-started/react-essentials) 같이 Suspense가 가능한 프레임워크를 사용한 데이터 가져오기

Reviewer: ```suggestion
- [Relay](https://relay.dev/docs/guided-tour/rendering/loading-states/)와 [Next.js](https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming#streaming-with-suspense) 같이 Suspense가 가능한 프레임워크를 사용한 데이터 가져오기
```

아래 삭제된 Next.js 링크를 반영해야 합니다. 기존 링크는 아마 깨질겁니다.

### 중첩된 콘텐츠가 로드될 때 보여주기 {/*revealing-nested-content-as-it-loads*/}

> 컴포넌트가 일시 중단되면 가장 가까운 상위 Suspense 컴포넌트가 fallback을 보여줍니다. 이를 통해 여러 Suspense 컴포넌트를 중첩하여 로딩 순서를 만들 수 있습니다. 각 Suspense의 fallback은 다음 레벨의 콘텐츠를 사용할 수 있게 되면 채워집니다. 예를 들어 앨범 목록에 자체 fallback을 지정할 수 있습니다.

Reviewer: ```suggestion
컴포넌트가 일시 중단되면 가장 가까운 상위 Suspense 컴포넌트가 Fallback을 보여줍니다. 이를 통해 여러 Suspense 컴포넌트를 중첩하여 로딩 순서를 만들 수 있습니다. 각 Suspense의 Fallback은 다음 레벨의 콘텐츠를 사용할 수 있게 되면 채워집니다. 예를 들어 앨범 목록에 자체 Fallback을 지정할 수 있습니다.
```

![image](https://github.com/user-attachments/assets/e98d4f67-f8c1-42f8-aace-e4f50506e60c)

위 규칙에 따라 React 관련 용어들은 모두 대문자로 시작하여야 합니다. 추가로, 아래 다른 모든 코드에 동일하게 적용해야 합니다.

`fallback` -> `Fallback`.

Author: 답변 감사드립니다!
문서의 앞 부분에서는 fallback으로 표기되어 있는데 해당 부분도 수정되어야 할까요?

Reviewer: 아 제가 https://github.com/reactjs/ko.react.dev/pull/1120 에서 지금 뒷쪽부터 문서 일괄적으로 수정해나가고 있긴 한데, 아직 거기까진 반영이 안됐네요ㅠㅠ

바꿔주시면 감사할 것 같습니다!

Author: 넵 알겠습니다!
답변 감사드립니다.
